core/templates/domain/state/StateObjectFactory.ts

```diff
@@ -167,29 +167,31 @@ export class StateObjectFactory {
     return constants.NEW_STATE_TEMPLATE as StateBackendDict;
   }
 
-  createDefaultState(newStateName: string): State {
+  createDefaultState(newStateName: string | null): State {
```

Comment:
Explanatory comment for the `null` is missing.

Reply:
Add TODO: Remove the createDefaultState so that full state can be created from start. No need to null then.